admin_guide/backup_restore.adoc

```diff
@@ -320,6 +320,15 @@ events. If the backup is still larger than 700 MB, stop the other hosts before
 performing this step.
 ====
 +
+[WARNING]
```

Comment:
Side note: can you update the TTL text above? I doubt this has anything to do with voltage levels in this case. "Time to live" is a better translation I guess ;-)